velox/functions/sparksql/MakeTimestamp.cpp

```diff
@@ -122,15 +123,20 @@ class MakeTimestampFunction : public exec::VectorFunction {
       if (args[6]->isConstantEncoding()) {
         auto tz =
             args[6]->asUnchecked<ConstantVector<StringView>>()->valueAt(0);
-        auto constantTzID = util::getTimeZoneID(std::string_view(tz));
+        const auto constantTzID =
+            util::getTimeZoneID(std::string_view(tz), false);
+        if (constantTzID == -1) {
+          context.setErrors(rows, makeInvalidTimezoneException());
```

Comment:
Perhaps, include timezone string in the error message to help troubleshooting.

Reply:
Changed to use the error message from getTimeZoneID which includes the timezone name.